### PR TITLE
Added Github Actions Daily Cron Job

### PR DESCRIPTION
**Pull request by adrianviquez**
- Created standards.yml file to run bash scripts for individual standards
- Deleted travis.yml file
- Added git username and email to bash scripts

### CODE REVIEW

File: scripts/standard-cpp-11.sh [hunk 1]
```diff
@@ -22,7 +22,8 @@ change_readme() {
 #commit files and upload to github repo with new commmit sha
 echo "Committing changes, pushing to GH repo..."
 commit_and_upload() {
-  
+  git config --global user.email "devops@codecov.local"
```
Comment (hootener) [inline scripts/standard-cpp-11.sh]: why `.local` here? instead of just `.io`?

File: scripts/standard-go.sh [hunk 2]
```diff
@@ -22,7 +22,8 @@ change_readme() {
 #commit files and upload to github repo with new commmit sha
 echo "Committing changes, pushing to GH repo..."
 commit_and_upload() {
-  
+  git config --global user.email "devops@codecov.local"
```
Comment (hootener) [inline scripts/standard-go.sh]: Same as above with .local --> .io

**Review by hootener — CHANGES_REQUESTED** [review]
I feel like we should change this to use the actual devops@codecov.io email, but if there's a good reason to keep it .local, I'll re-review and approve this PR.

**Comment by adrianviquez** [comment]
No particular reason I don't think, that's the email I was provided with by @drazisil-codecov for previous runs but I don't think it makes a difference. GH only complains if there isn't a username and email. I'll make the changes to use devops@codecov.io

File: scripts/standard-cpp-11.sh (hunk 1 above)
Comment (adrianviquez) [reply scripts/standard-cpp-11.sh]: Made changes on newest commit

File: scripts/standard-go.sh (hunk 2 above)
Comment (adrianviquez) [reply scripts/standard-go.sh]: Made changes on newest commit

**Review by hootener — APPROVED** [review]

**Pull request merged**